TestContainer/test/testReleaseNotes.js

```diff
@@ -4,18 +4,106 @@ const chaiHttp = require("chai-http");
 const { expect } = require("chai");
 
 chai.use(chaiHttp);
+
 var accessToken;
-const addressDelete = "/api/releasenote/1";
+const addressGet = "/api/releasenote";
+const addressGetSingle = "/api/releasenote/5";
+const addressGetSingleFail = "/api/releasenote/23123";
+const addressPut = "/api/releasenote/100";
+const addressPutFail = "api/releasenote/123123125";
+const addressDelete = "/api/releasenote/5";
 const addressDeleteFail = "/api/releasenote/897324";
 
+var noteToDelete;
+
 const init = () => {
   accessToken = TokenHandler.getAccessToken();
 };
 
+describe("Release notes GET", () => {
+  before(() => init());
+  // Should return a 401 unauth
+  it("GET | Get all notes without auth", done => {
+    chai
+      .request(process.env.APP_URL)
+      .get(addressGet)
+      .end(err => {
+        err.should.have.status(401);
+        done();
+      });
+  });
+
+  // should return 200 OK and a array of release notes
+  it("GET | Returns all the release note dummy data", done => {
+    chai
+      .request(process.env.APP_URL)
+      .get(addressGet)
+      .set("Content-Type", "application/json")
+      .set("Authorization", "Bearer " + accessToken)
+      .end((err, res) => {
+        if (err) {
+          done(err);
+        }
+        res.should.have.status(200);
+        done();
```

Comment:
Legg til sjekk for alle felter vi har definert.

Reply:
Ikke så lurt å sjekk på statisk array-lengde på 5. Foreslår at du fjern akkurat den testen, også test du heller at feltan på den første itemen i arrayen.